src/visual.ts

```diff
@@ -782,15 +769,16 @@ export class DotPlot implements IVisual {
             );
         });
 
-        if (this.settings.categoryAxis.show) {
+        if (this.formattingSettings.categoryAxis.show.value) {
             // Should handle the label, units of the label and the axis style
             xAxisProperties.axisLabel = this.data.categoryAxisName;
         }
 
         this.xAxisProperties = xAxisProperties;
     }
 
-    private renderAxis(duration: number): void {
+    // TODO: add animation duration
```

Comment:
what about this plans?

Reply:
It wasn't implemented previously. If we add the animation then we should add it not only for axis but for the whole visual. I'll be honest, I'm not sure how to implement animation and if we should even do it.